evm/src/executor/fork/backend.rs

```diff
@@ -396,19 +392,17 @@ impl Drop for SharedBackend {
     fn drop(&mut self) {
         // disconnect the command channel
         self.backend.disconnect();
-        if self.backend.is_closed() {
-            tracing::trace!( target: "sharedbackend", "shutting down");
-            // was the last sender, let the handler know and wait until it gracefully shut down
-            let (ack, rx) = oneshot_channel();
-
-            if self.shutdown.try_send(ack).is_ok() {
-                tracing::trace!( target: "sharedbackend", "waiting for ack");
-                if let Err(err) = rx.recv() {
-                    warn!(target: "sharedbackend", "Failed to receive ack:{}", err);
-                }
+        tracing::trace!( target: "sharedbackend", "shutting down");
+        // was the last sender, let the handler know and wait until it gracefully shut down
```

Comment:
without that removed `if`, this comment is not true

Reply:
you're right, missed that one,
wanna open a PR that removes that?